tests/kafkatest/tests/quota_test.py

```diff
@@ -76,7 +76,7 @@ def min_cluster_size(self):
     def run_clients(self, producer_id, producer_num, consumer_id, consumer_num):
         # Produce all messages
         producer = ProducerPerformanceService(
-            self.test_context, producer_num, self.kafka, security_protocol=self.security_protocol,
+            self.test_context, producer_num, self.kafka,
```

Comment:
since you've removed `security_protocol` from the `ProducerPerformanceService` and `ConsumerPerformanceService` constructor calls, you  can remove `self.security_protocol` and `self.interbroker_security_protocol` on lines 53 and 54